README.md

```diff
@@ -1,40 +1,55 @@
+# Azure Sentinel
+In the world of technology, cybersecurity is the #1 priority for any company. And with the almost infinite amount of threats any company can face, it seems almost impossible to do. In addition to that, there are many companies that are selling partial solutions but it gets cumbersome to make sure which version of what works with each other. It woulds be easier if all of the solutions were in one service. Microsoft heard that call and answered that call with Azure Sentinel.
 
-Azure Sentinel is a SIEM (Security Event Information Management) and a SOAR Security Orchestrated Response) that Microsoft has created. Azure Sentinel  delivers security and threat intelligence and proving a single solution for alert detection, threat visibility, proactive hunting and threat response
+Azure Sentinel is both a SIEM (Security Information and Event Management) and a SOAR (Security Orchestration, Automation, and Response) solution that Microsoft has created. Azure Sentinel delivers security and threat intelligence and proving a single solution for alert detection, threat visibility, proactive hunting, and threat response.
 
-## Data Connectors 
-With Azure Sentinel you can connect all your data sources with data connectors. Sentinel comes with multiple built in connections for Microsoft solutions. For example, you can connect Office 365 for real time integration
+## Data Connectors
+With Azure Sentinel, you can connect all your data sources with data connectors. Sentinel comes with multiple built-in connections for Microsoft solutions. For example, you can connect Office 365 for real time integration.
+
+## Workbooks
+Once you connect your resources to Azure Sentinel, you can monitor the data. You can use the built-in Workbooks that Azure provides or you can make custom workbooks. You can build dashboards from the workbooks to visualize the data within the Azure Portal.
 
-## Workbooks 
-Once you connect you resources to Azure Sentinel you can monitor the data. You can use the built in Workbooks that Azure provides or you can make custom workbooks. You can build dashboards from the workbooks to visualize the data within the Azure Portal. 
 ![](images/workbooks.PNG)
+
 ## Analytics
-Analytics are rules that can help minimize that amount of threats that have to be reviewed and investigated. You can use Analytics to create Incidents. Analytics ranks severity from high to medium to low to informational. Azure Sentinel will automatically rank the rules them from most sever to the least severe. Below is an example of what Analytics can look in aby companies workspace
+Analytics are rules that can help minimize that amount of threats that have to be reviewed and investigated. You can use Analytics to create Incidents. Analytics ranks severity from high to medium to low to informational. Azure Sentinel will automatically rank the rules from most severe to the least severe. Below is an example of what Analytics can look like in any company's workspace.
```

Comment:
Is "Analyics" the name of the actual service within Sentinel? just asking because that's how it comes across. Analytics is also a term for more general processing that occurs on data, so you want to make sure you're being clear.